### PR TITLE
Wikipedia Fearure Added

### PR DESCRIPTION
**Pull request by Abhi13e**
Wikipedia feature added. We have to speak "wikipedia (anything you want to get information)" afterward it will give  information in audio and save in conversation.txt.

### CODE REVIEW

**Comment by Abhi13e** [comment]
@TobiasDemuth Hey have you review my code? ..... is anything required to change more?

File: speech_assistent.py [hunk 1]
```diff
@@ -221,7 +226,18 @@ def personal_assistant():
                 translation = offline_translate(text,input_lang)
                 print(translation)
                 response = translation
-            
+                
+            if key in user_input.lower() and any(command in user_input.lower() for command in wikipedia_commands):
+                search_query = user_input.lower().replace(key, "").strip()
+                response = get_wikipedia(search_query)
+                with open("conversation.txt", "a") as file:
```
Comment (TobiasDemuth) [inline speech_assistent.py:233]: conversation writing is handled at the end of the file so it can be deleted

**Review by TobiasDemuth — CHANGES_REQUESTED** [review]
so delete the conversation handling you added and adjust input handling like the comment sais the rest looks just fine :)

**Review by TobiasDemuth — APPROVED** [review]

**Comment by Abhi13e** [comment]
Hey @TobiasDemuth i need to change or you already make changes in that?

**Comment by TobiasDemuth** [comment]
@Abhi13e already changed no worries :)

**Pull request merged**